.changelog/3848.breaking.md

```diff
@@ -0,0 +1 @@
+runtime: Add support for block tags
```

Comment:
Why is this breaking?

Reply:
Hm i was thinking to mark it as breaking as `ExecuteTxResult` (publicly exposed runtime api field) was changed. I guess it depends on what definition of "breaking" we'd like to use for these fragments :-)

Reply:
Oh, yeah that's true, keep it as breaking.